vue2/src/store/index.js

```diff
@@ -1,10 +1,41 @@
 import { createStore } from 'vuex'
-
+import { years } from '../views/Home'
 export default createStore({
   state: {
-    // message:'lalala'
+    sex: '',
+    year: years[0].year,
+    month: '1',
+    day: '1',
+    page2_1:'',
```

Comment:
前回ご指摘させていただいておりますが、こちらもスペースが必要です。
```suggestion
    page2_1: '',
```

スペースがある所と無い所がチラつくと、わかっているのに徹底されていないという少し雑な記述の印象を与えかねませんので気をつけましょう。
これもフォーマット自動整形で対応できます。